kong/plugins/rate-limiting/handler.lua

```diff
@@ -29,6 +29,18 @@ local function get_identifier(conf)
     end
   elseif conf.limit_by == "credential" then
     identifier = ngx.ctx.authenticated_credential and ngx.ctx.authenticated_credential.id
+  elseif conf.limit_by == "key" then
+    identifier = kong.request.get_headers()[conf.key_name] or kong.request.get_query()[conf.key_name]
+
+    if not identifier and conf.key_in_body then
+      local body, err = kong.request.get_body()
+
+      if err then
+        ngx_log(ngx.ERR, "Cannot process request body: ", tostring(err))
```

Comment:
Please write the above as:
```lua
local body, err = kong.request.get_body()
if err then
  ngx_log(...)
  return
end

identifier = ...
```
 
As per our style guide and rules around early-exit branches. Also, error logs should _not_ be capitalized. Thanks!

Reply:
Right. I should have expanded the diff! Nonetheless, we should avoid the empty line between the call and the error handling :)